### DISCUSSION TITLE
I don't know why templater occurs this parsing error

### DISCUSSION THREAD

**Discussion by KK-3458** [SilentVoid13/Templater#770] [Help]
I want to use calendar to create a new daily everyday include tomorrow's diary. The template of this diary uses the syntax of templater, but it is not successful. I don't know why, the following error occurred:
<img width="275" alt="image" src="https://user-images.githubusercontent.com/66560506/179397823-0dff4d46-bd31-4f0d-818b-ee517afe1ae3.png">
then I check the console for more detail, it is as following:
<img width="311" alt="image" src="https://user-images.githubusercontent.com/66560506/179397866-158f8506-dafa-49c1-91ee-3d40dbc127dd.png">
I don't know why, who can help me please!

**Comment by AB1908** [comment]
Show us the template

**Comment by KK-3458** [comment]
Now I know where the error is. My template is modified according to others' templates, and its original template has custom variables, but I deleted the fragment of custom variables, but kept the fields that need to use the variables, so I made an error, so I just added the functions defined in the original template😅

**Comment by AB1908** [comment]
Sounds good, glad you were able to resolve this.
